src/Core/Resolvers/CosmosMutationEngine.cs

```diff
@@ -65,13 +65,23 @@ private async Task<JObject> ExecuteAsync(IMiddlewareContext context, IDictionary
             string entityName = metadataProvider.GetEntityName(graphQLType);
             AuthorizeMutation(context, queryArgs, entityName, resolver.OperationType);
 
-            ItemResponse<JObject>? response = resolver.OperationType switch
+            JObject result;
+            if (resolver.OperationType == EntityActionOperation.Patch)
             {
-                EntityActionOperation.UpdateGraphQL => await HandleUpdateAsync(queryArgs, container),
-                EntityActionOperation.Create => await HandleCreateAsync(queryArgs, container),
-                EntityActionOperation.Delete => await HandleDeleteAsync(queryArgs, container),
-                _ => throw new NotSupportedException($"unsupported operation type: {resolver.OperationType}")
-            };
+                result = await HandlePatchAsync(queryArgs, container);
```

Comment:
why is patch response different? is there no "result.Resource" for patch response? is it because it's batch patch operation per https://learn.microsoft.com/en-us/azure/cosmos-db/partial-document-update-getting-started?tabs=dotnet#prerequisites:~:text=Combine%20multiple%20patch%20operations%3A

Reply:
Because `TransactionalBatchResponse` does not return `Itemresponse` like other mutation operations.